pkg/restapi/issuer/operation/operations.go

```diff
@@ -155,6 +156,7 @@ type Config struct {
 	ExternalURL          string
 	DidDomain            string
 	JSONLDDocumentLoader ld.DocumentLoader
+	CmOutputDescriptor   map[string][]*cm.OutputDescriptor
```

Comment:
this object will be heavy, I would suggest you put a helper struct here which has functions to find and return subset of output descriptors by credential scope. So that you can swap underlying memory store with cache in future with little effort.

Reply:
#580 I will address this optimization in the follow up pr , if its okay with you @sudeshrshetty